tenv.go

```diff
@@ -211,3 +220,35 @@ func checkSelectorExprTarget(typ *ast.SelectorExpr) bool {
 	targetName := x.Name + "." + typ.Sel.Name
 	return targetName == "testing.TB"
 }
+
+// goVersionLower117 returns true if version is lower than go1.17.
+// In case of any parse errors it returns false.
+func goVersionLower117(version string) bool {
+	version = strings.TrimPrefix(version, "go")
+	if version == "" {
+		return false
+	}
+
+	parts := strings.Split(version, ".")
+	if len(parts) != 2 {
+		return false
+	}
+
+	major, err := strconv.Atoi(parts[0])
+	if err != nil {
```

Comment:
Why ignore this error ? It's unexpected input, so we should report correct input.